packages/nestjs/README.md

```diff
@@ -38,6 +38,23 @@ Sentry.init({
 
 Note that it is necessary to initialize Sentry **before you import any package that may be instrumented by us**.
 
+## Span Decorator
+
+Use the @GetSentrySpan() decorator to gain additional performance insights for any function within your NestJS application.
```

Comment:
l: Is `GetX` a pattern that is common for nest? To me it reads a bit weird, something like `WrapWithSentrySpan` or something along these lines would feel more explanatory to me, but no strong feelings!

Reply:
SentryTraced

Reply:
No, GetX is not super common in nest was just an initial idea. I like SentryTraced if everyone is happy with that

Reply:
Updated to SentryTraced